spec/user_spec.rb

```diff
@@ -0,0 +1,14 @@
+describe User do
+  let!(:user) { User.create( email: 'user@email.com',
```

Comment:
Style/BlockDelimiters: Avoid using `{...}` for multi-line blocks. (https://github.com/bbatsov/ruby-style-guide#single-line-blocks)<br>Style/MultilineBlockLayout: Block body expression is on the same line as the block start.